docs/content/docs/install/settings.md

```diff
@@ -50,3 +50,13 @@ There is a few things you can configure through the configmap
 
   This will provide us to give extra IPS (ie: 127.0.0.1) or networks (127.0.0.0/16)
   separated by commas.
+
+* `max-keep-run-upper-limit`
+
+  This allows user to define a global max limit for max-keep-run value. If user has defined annotation on pipelineRun,
```

Comment:
It would be great if you mention  to which resource max-keep-run value is specified